Content.Server/BluespaceHarvester/BluespaceHarvesterRiftComponent.cs

```diff
@@ -0,0 +1,53 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.BluespaceHarvester;
+
+[RegisterComponent]
+public sealed partial class BluespaceHarvesterRiftComponent : Component
```

Comment:
It's probably not worth creating new systems and components here, either. There is already a dragon rift in the game - you could edit it to add the ability to maintain both the old behavior and the harvester rift behavior.

Reply:
I don’t really think so, there’s a code for buying points, and I also don’t need the growth of the portal itself. As a result, we get that through datafields we switch the operation of our prototype.